src/ontology/mondo-edit.obo

```diff
@@ -617799,6 +617814,83 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/22219 ! KIAA1549
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040084
+name: NDP-related vitreoretinopathy including Norrie syndrome
+def: "Any vitreoretinopathy caused by a variant in the NDP gene, including cases diagnosed as Norrie disease or X-linked exudative vitreoretinopathy 2." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "NDP-related vitreoretinopathy including Norrie syndrome'" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0020246 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited vitreoretinopathy
+intersection_of: MONDO:0020246 ! inherited vitreoretinopathy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/7678 ! NDP
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040085
+name: CDHR1-related retinopathy
+def: "Any retinopathy caused by variants in the CDHR1 gene, including cases diagnosed as cone-rod dystrophy 15, retinal macular dystrophy, or retinitis pigmentosa 65." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "CDHR1-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14550 ! CDHR1
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040086
+name: GUCA1A-related retinopathy
+def: "Any retinopathy caused by a variant in the GUCA1A gene, including cases diagnosed as cone dystrophy 3 or cone-rod dystrophy 14." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
```

Comment:
I don't like "including cases diagnosed as ~"
The definition refers to a "retinopathy". A retinopathy does not include "cases diagnosed as".
Please rephrase.